mahout/src/main/java/org/apache/zeppelin/mahout/MahoutSparkInterpreter.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.mahout;
+
+import java.net.URL;
+import java.util.*;
+
+import org.apache.spark.repl.SparkIMain;
+import org.apache.zeppelin.interpreter.*;
+import org.apache.zeppelin.spark.*;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.apache.zeppelin.mahout.utils.*;
+
+/**
+ * Spark interpreter for Zeppelin.
+ *
+ */
+
+
+public class MahoutSparkInterpreter extends SparkInterpreter {
+
+  private static final Logger logger = LoggerFactory.getLogger(MahoutSparkInterpreter.class);
+  private SparkIMain intp;
+  private boolean firstRun = true;
+
+  static {
+    Interpreter.register(
+      "spark",
+      "mahout",
+      MahoutSparkInterpreter.class.getName(),
```

Comment:
Fought for an entire day trying to implement new dynamic terp loading protocols, but couldn't get it to register.  Left *.json as artifact, maybe we can get it working right before merge- any hints what I may be doing wrong?